### PR TITLE
eelsmodel raise ValueError when no microscope metadata

### LINKED ISSUE #1427: Creating EELSmodel fails with minimal signal
```Python
s = hs.signals.EELSSpectrum(range(100))
s.add_elements(("O","Fe"))
m = s.create_model()
```

Gives the error https://gist.github.com/magnunor/3a8a6f7fefbbb3e62463dda9a6138870

### PR DESCRIPTION
**Pull request by francisco-dlp**
Fixes #1427

### CODE REVIEW

File: hyperspy/gui/__init__.py [hunk 1]
```diff
@@ -41,4 +41,7 @@ def set_ets_toolkit(toolkit):
 elif "Qt" in backend:
     set_ets_toolkit("qt4")
 else:
+    print("here")
```
Comment (magnunor) [inline hyperspy/gui/__init__.py]: Should this `print` be here?

Comment (francisco-dlp) [reply hyperspy/gui/__init__.py]: Fixed, thanks!

**Comment by magnunor** [comment]
Looks good!

**Comment by magnunor** [comment]
I'll cherry-pick the commits into RELEASE_next_minor

**Comment by francisco-dlp** [comment]
We should merge RELEASE_next_patch into RELEASE_next_minor instead and modify as required if any nosetest got in there.

**Comment by magnunor** [comment]
Like this: https://github.com/hyperspy/hyperspy/pull/1434 ?

I saw one nosetest in there, I'll change it to pytest

**Pull request merged**